### PR TITLE
feat: role & permissions

### PR DESCRIPTION
**Pull request by pemudakoding**

### CODE REVIEW

File: database/seeders/PermissionsSeeder.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Enums\Permission;
```
Comment (holiq) [inline database/seeders/PermissionsSeeder.php:5]: I think this enum will be conflict into the Permission model

File: database/seeders/PermissionsSeeder.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Database\Seeder;
+use Spatie\Permission\Models\Permission;
```
Comment (holiq) [inline database/seeders/PermissionsSeeder.php]: on this model, you can use an alias for fix it

**Review by holiq — CHANGES_REQUESTED** [review]
i think just that the problem

File: database/seeders/PermissionsSeeder.php (hunk 2 above)
Comment (pemudakoding) [reply database/seeders/PermissionsSeeder.php]: good catch mate

**Review by holiq — APPROVED** [review]

**Pull request merged**